src/Chainweb/Chainweb.hs

```diff
@@ -270,7 +275,9 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configMempoolP2p = defaultEnableConfig defaultMempoolP2pConfig
     , _configPruneChainDatabase = True
     , _configBlockGasLimit = 100000
-    , _configCutFetchTimeout = 3000000
+    , _configCutFetchTimeout = 10000000
```

Comment:
Attention

Reply:
Yes, is this still needed now that the catchup step size is reduced?